semantics/language/common/expr/bitwise.k

```diff
@@ -56,21 +56,21 @@ module C-COMMON-EXPR-BITWISE
           [structural]
      rule (.K => UNDEF("CEB2",
                "the right operand in a bitwise shift is negative.",
-               "6.5.7:3"))
+               "6.5.7:3 J.2:1 item 51"))
```

Comment:
Until now you used a comma to separate the sections/anexes.  Now you are using space.